frontend/src/app/main/data/workspace.cljs

```diff
@@ -260,12 +260,11 @@
 
     ptk/WatchEvent
     (watch [_ state _]
-      (let [ignore-until  (-> state :workspace-file :ignore-sync-until)
+      (let [file-data     (-> state :workspace-data)
+            ignore-until  (-> state :workspace-file :ignore-sync-until)
```

Comment:
Is this better than 
file-data (:workspace-data state)
ignore-until (get-in state [:workspace-file :ignore-sync-until])
? Because I think we use a lot more the get-in form...

Reply:
In the first case I'm not really sure.

In the second, actually get-in is slower because it receives a dynamic vector of arguments, and the compiler optimizer cannot convert it to static accessors. But we have a `(d/get-in` macro that solves this. So I'm not either sure if this construct is better to d/get-in.

Reply:
The `->` is faster than using `get-in`, if you are going to change `->` to get-in, please, use `dm/get-in` (faster, macro based version)

Reply:
> In the first case I'm not really sure.
> 
> In the second, actually get-in is slower because it receives a dynamic vector of arguments, and the compiler optimizer cannot convert it to static accessors. But we have a `(d/get-in` macro that solves this. So I'm not either sure if this construct is better to d/get-in.

dm/get-in is analogous to -> performance

Reply:
changed